dist/profile/templates/confluence/vhost.conf

```diff
@@ -1123,6 +1123,7 @@ RewriteRule "^/display/JENKINS/Results\+Cache\+Plugin$" "https://plugins.jenkins
 RewriteRule "^/display/JENKINS/Reverse\+Proxy\+Auth\+Plugin$" "https://plugins.jenkins.io/reverse-proxy-auth-plugin" [NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Rich\+Text\+Publisher\+Plugin$" "https://plugins.jenkins.io/rich-text-publisher-plugin" [NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Rigor\+Optimization\+Plugin$" "https://plugins.jenkins.io/rigor-optimization" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Role\+Strategy\+Plugin$" "https://plugins.jenkins.io/role-strategy" [NC,L,QSA,R=301]
```

Comment:
aren't all plugins already covered by a different rule?

https://wiki.jenkins.io/display/JENKINS/Role+Strategy+Plugin

already redirects

Reply:
Thanks for detecting that.  There is already a line that redirects the role strategy plugin.  We can resolve #3736 without needing to merge this pull request because it is already implemented in the file.